src/components/Whiteboard/WhiteboardHost.tsx

```diff
@@ -113,8 +113,9 @@ export const WhiteboardHostConnected = () => {
   const { state: presentationState } = usePresentationMode();
   const isPresenting = presentationState.type === 'presenting';
   const isViewingPresentation = presentationState.type === 'presentation';
-  const isViewingPresentationInEditMode =
-    isViewingPresentation && presentationState.isEditMode;
+
+  const isEditModeEnabled =
+    presentationState.type === 'idle' || presentationState.isEditMode;
```

Comment:
This sounds as better name of this condition that it is used in `CollaborationBar` as mentioned above, maybe `isEditEnabled` would be better because it means not only edit mode in presentation but edit possibility when no presentation is ongoing.